### PR TITLE
Get rid of unnecessary “test” environment

### PR DESCRIPTION
**Pull request by iamigo**
(was only being used for view tests)—just use the “build” env for this

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -13,7 +13,6 @@
  */
 'use strict'; // eslint-disable-line strict
 require('./config/toggles'); // Loads the feature toggles
```
Comment (annyhe) [inline config.js]: @iamigo Do we need this line? Seems like we are not using toggles in this file.

Comment (iamigo) [inline config.js]: Yeah I saw that too--I meant to take it out in this PR... I'll catch in the next one.

Comment (iamigo) [inline config.js]: Actually having this line here makes sure that the feature flags are loaded *before* the config is loaded.

Comment (annyhe) [inline config.js]: Do we need the feature flags in this module? Where are we using it?

**Pull request merged**